src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -441,12 +441,11 @@
     examples:
         - name: Import from the same registry.
           text: >
-            az acr image import -n registry --source-image registry.azurecr.io/repository:tag -t repo:tag .
+            az acr image import -n registry --source-image registry.azurecr.io/repository:tag -t targetRepository:targetTag
```

Comment:
Still not sure about this string `targetRepository:targetTag` ... Basically repository doesn't allow upper case. Let me know what you think. /cc @NatalieGong

Reply:
I can make it lower case if you prefer.

Reply:
definitely don't put non-working examples in the examples area 👍